src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            StringBuilder stringBuilder = new StringBuilder();
+            int supply = 0;
+            int buy = 0;
+            int result = 0;
+            for (String readAllLine : Files.readAllLines(new File(fromFileName).toPath())) {
+                String[] split = readAllLine.split(",");
+                int parseInt = Integer.parseInt(split[1]);
+                if (readAllLine.contains("supply")) {
+                    supply += parseInt;
+                } else {
+                    buy += parseInt;
+                }
+                result = supply - buy;
+            }
+            String s = String.valueOf(supply);
+            String b = String.valueOf(buy);
```

Comment:
we don't need this conversion, append will work when you simply pass the value 
`.append(supply)`